### PR TITLE
Vagrant and CI tests for Ubuntu 22.04

### PR DESCRIPTION
**Pull request by mtmail**
Ubuntu 22 LTS got released https://wiki.ubuntu.com/Releases

The `ci-tests.yml` still have some mentions of CentOS, I wasn't sure if we want to remove them as well.

### CODE REVIEW

**Comment by lonvia** [comment]
You need to make the script executable.

> The ci-tests.yml still have some mentions of CentOS, I wasn't sure if we want to remove them as well.

Just leave them, I haven't made up my mind about #2601 yet.

**Comment by mtmail** [comment]
Somewhere while running `Purging configuration files for postgresql-common (238.pgdg20.04+1) ...` for Ubuntu 22 there is a dependency failure related to R. https://cloud.r-project.org/bin/linux/ubuntu/fullREADME.html doesn't have Ubuntu 22 packages released yet. I'd say we wait a bit longer instead of trying to add work-arounds.

**Comment by lonvia** [comment]
Oh, did this include the tests already? I don't think it's worth bothering before there is an official Ubuntu 22 Github Actions environment. But I see that the installation run works fine. So lets keep that.

**Comment by mtmail** [comment]
Github Actions has Ubuntu 22 in pubic beta now https://github.com/actions/virtual-environments/issues/5490

**Comment by lonvia** [comment]
Oh lovely, phpunit has breaking changes, too.

Repeating my recommendation from above: let's get the installation instruction + CI Tests / install merged. Then tackle all the failing tests and checks one after another in separate PRs.

**Comment by mtmail** [comment]
https://github.com/osm-search/Nominatim/pull/2706 deals with the phpunit failures. Tested locally on Ubuntu 20 and 22.

File: .github/workflows/ci-tests.yml [hunk 1]
```diff
@@ -158,11 +168,6 @@ jobs:
 
             - name: Install test prerequsites
               run: sudo apt-get install -y -qq python3-behave
-              if: matrix.ubuntu == 20
-
-            - name: Install test prerequsites
-              run: pip3 install behave==1.2.6
-              if: matrix.ubuntu == 18
```
[the remark this answers is not in the document]
Comment (mtmail) [reply .github/workflows/ci-tests.yml:165]: Must have been a mistake.

**Comment by wahyu9kdl** [comment]
## Operating System
 Ubuntu
 20.04.4
 LTS
 ## [AWDEV]

https://devoloper.awdev.eu.org
https://wahyu9kdl.github.io

 ## Virtual Environment
 - Environment: ubuntu-20.04
 - Version: 20220515.1
 - Included Software: https://github.com/actions/virtual-environments/blob/ubuntu20/20220515.1/images/linux/Ubuntu2004-Readme.md
- Image Release: https://github.com/actions/virtual-environments/releases/tag/ubuntu20%2F20220515.1

Reported success!
 
 Evaluate and set environment url
 Evaluated environment url: https://wahyu9kdl.github.io/
 Cleaning up orphan processes

[DOWNLOD LOG](https://github.com/wahyu9kdl/wahyu9kdl.github.io/suites/6695385341/logs?attempt=1)

[GET STARTED](https://wahyu9kdl.github.io/DASHBOARD/TOOLS/CodeSandbox.html)

[Tes Website Responsive](https://wahyu9kdl.github.io/DASHBOARD/TOOLS/website-responsive-tools.html)

[VISITOR BADGE](https://wahyu9kdl.github.io/HTML/TOOLS/visitor-badge.html)

**Pull request merged**